.travis.yml

```diff
@@ -15,6 +15,7 @@ script:
   - docker run -a stdout -a stderr --net host --rm --privileged --name build-machine --mount type=bind,source="$(pwd)",target=/go/src/github.com/kuzzleio/sdk-go kuzzleio/sdk-cross:amd64 /build.sh
   - sh .codepipeline/start_kuzzle.sh
   - docker run -a stdout -a stderr --privileged --rm --network codepipeline_default --link kuzzle --name test-machine --mount type=bind,source="$(pwd)",target=/go/src/github.com/kuzzleio/sdk-go kuzzleio/sdk-cross:amd64 /test.sh
+  - docker run --rm -it --mount type=bind,source="$(pwd)",target=/go/src/github.com/kuzzleio/sdk-go kuzzleio/sdk-cross:android386 /build.sh
```

Comment:
why "386" and not x86?
and what about arm and arm64?

Reply:
386 and x86 are the same?
arm and arm64 will be handled with the CI, this is just a first draft where we successfully compile the android SDK, there is more work to do to package the final AAR file which will embed x86, armeabi-v7a and armeabi-v8a

Reply:
No, 386 and x86 are not the same: x86 refer to 8086 CPUs, from 286 CPUs to to modern pentiums, which are also x64 (aka x86-64).
On the other hand, "386" refer only to CPU from the late eighties ;-)

Reply:
:+1: 
I feel old now..